content/desktop/release-notes.md

```diff
@@ -25,6 +25,74 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.30.0
+
+{{< release-date date="2024-05-06" >}}
+
+{{< desktop-install all=true version="4.30.0" build_path="/TBD/" >}}
+
+### New
+
+#### For all platforms 
+
+- Added additional methods for administrators to enforce sign-in.
+- Docker Desktop now supports SOCKS5 proxies. Requires a Business subscription.
+- Improved extensibility via CLI hooks with new the Docker Engine Moby 26.1 release.
+- Added a new setting to manage the onboarding survey in [Settings Management](hardened-desktop/settings-management/_index.md)
+
+#### For Windows
+
+- Added support for Kerberos and NTLM proxy authentication on Windows, available Business subscription users.
```

Comment:
"available for Business subscription users"

or

"Requires Business subscription"